docs/components/dropdown.md

```diff
@@ -22,7 +22,7 @@ The dropdown component can be used to show a list of menu items when clicking on
 <fwb-dropdown-example-placement />
 ```vue
 <template>
-  <fwb-dropdown text="Bottom">
+  <fwb-dropdown text="Top">
```

Comment:
It also requires `placement="top"` to show on top... as in the example file here: `docs/components/dropdown/examples/FwbDropdownExampleAlignment.vue`